python/revng/cli/_commands/fetch_debuginfo/common.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import gzip
+import os
+import shutil
+import sys
+
+import requests
+
+
+def log(message, verbose):
```

Comment:
The `verbose` argument is effectively a global variable here.
Consider making a global variable that's private to the module.

Reply:
I've resolved this one, but I am having this ```check-convention``` error:

```
$ orc shell revng check-conventions --force-format python/revng/cli/_commands/fetch_debuginfo/__init__.py 
python/revng/cli/_commands/fetch_debuginfo/__init__.py:49: error: Name "common" is not defined
```
It is strange I guess. @mrjackv, you may help me with this? Thanks!